examples/seismic/acoustic/acoustic_example.py

```diff
@@ -99,4 +99,4 @@ def run(dimensions=(50, 50, 50), spacing=(20.0, 20.0, 20.0), tn=1000.0,
 
 
 if __name__ == "__main__":
-    run(full_run=False, autotune=False, space_order=6, time_order=2)
+    run(full_run=True, autotune=False, space_order=6, time_order=2)
```

Comment:
I can reduce dedault tn to keep it at ok level memorywise. All operators are now tested so this one is not really mandatory indeed so can be skipped in tests. Would still kepp it as example of our wrappers